### PR TITLE
flow accounts remove

### LINKED ISSUE #68: flow accounts remove
### Issue To Be Solved

Currently it is not possible to remove a contract via the flow accounts command

### (Optional): Suggest A Solution

Implement `flow accounts remove`

### (Optional): Context
https://docs.onflow.org/cadence/language/contracts#removing-a-deployed-contract

### PR DESCRIPTION
**Pull request by bjartek**
Closes #68 

There was no template in the flow sdk repo so I just made it inline here, I did not want to file a PR against two repos but can add the template in a seperate PR and refactor this afterwards?

### CODE REVIEW

**Comment by bjartek** [comment]
see https://github.com/onflow/flow-go-sdk/pull/152 for the template

**Review by janezpodhostnik — COMMENTED** [review]
Very nice addition! Thank you @bjartek.

**Comment by sideninja** [comment]
@bjartek  I think it's a great feature to support in CLI, however, if you don't mind waiting for the PR onflow/flow-go-sdk#152 to be closed and then implement this without the temp template fix.

**Comment by bjartek** [comment]
Sure

**Comment by janezpodhostnik** [comment]
Close was automatic. This shouldn't be closed

**Comment by psiemens** [comment]
@bjartek you can use your template in Flow Go SDK v0.15.0 now: https://github.com/onflow/flow-go-sdk/releases/tag/v0.15.0

**Comment by sideninja** [comment]
@psiemens as @janezpodhostnik pointed out this is blocked by emulator dependency (cadence 0.12.6) which depends on flow-go (cadence 0.13). Can we release a new version of the emulator so we can add it here?

**Comment by bjartek** [comment]
Not really sure why the go.mod is different here, I just ran go mod tidy before i committed.

**Review by janezpodhostnik — APPROVED** [review]

File: flow/accounts/remove-contract/remove.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	Host    string `flag:"host" info:"Flow Access API host address"`
+	Results bool   `default:"false" flag:"results" info:"Display the results of the transaction"`
+}
+
+var conf Config
+
+var Cmd = &cobra.Command{
+	Use:   "remove-contract <name>",
+	Short: "Remove a contract deployed to an account",
+	Args:  cobra.ExactArgs(2),
```
Comment (sideninja) [inline flow/accounts/remove-contract/remove.go:42]: Should be 1
```suggest
Args:  cobra.ExactArgs(1),
```

**Pull request merged**